kolibri/plugins/facility/assets/src/routes.js

```diff
@@ -124,4 +126,14 @@ export default [
       }
     },
   },
+  {
+    path: '/:facility_id?/managesync',
+    component: ManageSyncSchedule,
+    name: PageNames.MANAGE_SYNC_SCHEDULE,
+  },
+  {
+    path: '/:facility_id?/:device_id?/editdevice/:deviceId',
```

Comment:
Only need `deviceId` once in the URL! So can just have:
`path: '/:facility_id?/editdevice/:deviceId',`